plugin-taskbar/lxqttaskbutton.h

```diff
@@ -89,25 +98,40 @@ public slots:
     void setOrigin(Qt::Corner);
 
 protected:
-    void dragEnterEvent(QDragEnterEvent *event);
-    void dragLeaveEvent(QDragLeaveEvent *event);
+    virtual void dragEnterEvent(QDragEnterEvent *event);
+    virtual void dragLeaveEvent(QDragLeaveEvent *event);
+    virtual void dropEvent(QDropEvent *event);
     void mousePressEvent(QMouseEvent *event);
     void mouseReleaseEvent(QMouseEvent *event);
     void mouseMoveEvent(QMouseEvent *event);
-    void contextMenuEvent(QContextMenuEvent *event);
+    virtual void contextMenuEvent(QContextMenuEvent *event);
     void paintEvent(QPaintEvent *);
 
+    void setWindowId(WId wid) {mWindow = wid;}
+    virtual void arbitraryMimeData(QMimeData * mimedata);
+    virtual void draggingTimerTimeout() {activateWithDraggable();}
+    virtual QString acceptMimeData() const {return taskButtonMimeDataFormat();}
```

Comment:
What are you doing here? This just implicitly returns `QString("lxqt/lxqttaskbutton")`. Is there a reason these two methods can't go away in favour of a #define?